packages/framework/src/Model/Mail/MailTemplateSender/MailTemplateSenderInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Model\Mail\MailTemplateSender;
+
+use Shopsys\FrameworkBundle\Model\Mail\MailTemplate;
+
+interface MailTemplateSenderInterface
+{
+    /**
+     * @return string|null
+     */
+    public function getFormLabelForEntityIdentifier(): ?string;
```

Comment:
The null as a return value means "do not render any identifier field," but intuitively, I expected "render empty label"... 
Maybe we could talk about it, what do you think?

Reply:
yeah, let's discuss it, please